bay/images/Dockerfile.clamav

```diff
@@ -8,24 +8,33 @@ RUN apk add --no-cache -u --repository http://dl-cdn.alpinelinux.org/alpine/edge
   unrar \
   supervisor
 
+# Add clamav user
+RUN mkdir -p /var/lib/clamav && \
+  mkdir /usr/local/share/clamav && \
+  chown -R clamav:clamav /var/lib/clamav /usr/local/share/clamav /etc/clamav
+
 # Initial update of av databases
 RUN curl -sLo /var/lib/clamav/main.cvd http://database.clamav.net/main.cvd && \
   curl -sLo /var/lib/clamav/daily.cvd http://database.clamav.net/daily.cvd && \
   curl -sLo /var/lib/clamav/bytecode.cvd http://database.clamav.net/bytecode.cvd && \
   chown clamav:clamav /var/lib/clamav/*.cvd && \
-  mkdir /run/clamav
+  mkdir /run/clamav && \
+  chown -R clamav:clamav /run/clamav
+
+COPY --chown=clamav:clamav clamav/eicar.com /
+COPY --chown=clamav:clamav clamav/readiness.sh /
```

Comment:
Can we configure a pod readiness script for Lagoon?

Reply:
Just checked with amazee and while this is technically possible with Openshift, its not currently supported in lagoon on native kubernetes.